src/components/modals/StepConnectDevice.js

```diff
@@ -11,7 +11,7 @@ import DeviceMonit from 'components/DeviceMonitNew'
 
 type Props = {
   accountName?: string,
-  account?: Account,
+  account?: Account | null,
```

Comment:
the previous was correct to me. if you don't define `account: undefined` as defaultProps.